### PR TITLE
Basic tasking

### PR DESCRIPTION
**Pull request by leogermond**
Add a 240b tasking slide set  with a reduced content

Slides
+ Remove and simplify code examples
+ Reorder slides a bit (intro a bit larger)
+ Snappier sentences
+ Bold and code formatting for easier presentation
+ Slide on `delay`
+ No lab

Course
+ Add "b" alpha prefix description to README

### CODE REVIEW

File: courses/fundamentals_of_ada/240b_tasking.rst [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+
+-------------
+Delay keyword
+-------------
+
+- :ada:`delay` keyword part of tasking
+- Blocks for a time
+- Relative: Blocks for a :ada:`Duration`
+- Absolute: Blocks until a given :ada:`Calendar.Time` or :ada:`Real_Time.Time`
+
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/240b_tasking.rst:208]: Add note that "delay" is minimum - if you delay for 3.0 seconds, RT guarantees _at least_ 3 seconds

File: courses/fundamentals_of_ada/240b_tasking.rst [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+==========================
+Task and Protected Types
+==========================
+
+---------------
+Task Activation
+---------------
+
+* An instanciated task starts running when **activated**
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/240b_tasking.rst:226]: instantiated

File: courses/fundamentals_of_ada/240b_tasking.rst [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+
+    - Activated when **enclosing** declarative part finishes its **elaboration**
+
+* On the heap
+
+    - Activated **immediately** at instanciation
+
+.. code:: Ada
+    
+   task type T is [...]
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/240b_tasking.rst:237]: We haven't talked about tasks vs task types yet. To keep this "basic", update the previous examples to always use "task type T"

File: courses/fundamentals_of_ada/240b_tasking.rst [hunk 4]
```diff
@@ -0,0 +1,492 @@
...
+   end select;
+
+------------------------
+Protected Object Entries
+------------------------
+
+* **Special** kind of protected :ada:`procedure`
+* Several tasks can be waiting on the same :ada:`entry`
+* Only **one** will be re-activated when the barrier is **relieved**
+* May use a **barrier**, allowing call **only** on a boolean condition
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/240b_tasking.rst:405]: Switch lines 404 and 405 (describe barrier first, then what happens)

**Review by frank-at-adacore — CHANGES_REQUESTED** [review]

File: courses/fundamentals_of_ada/240b_tasking.rst (hunk 3 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/240b_tasking.rst:237]: Ok fair enough, and I'll add some note about `task T` implying the declaration of an anonymous task type with a named instance T.

**Review by frank-at-adacore — APPROVED** [review]

**Pull request merged**